src/main.rs

```diff
@@ -12,7 +12,7 @@ use std::{
 };
 
 fn main() -> Result<(), ProgramError> {
-    dotenv().expect("Failed to initialize environment");
+    optional_dotenv();
```

Comment:
```suggestion
    dotenv().ok();
    init_from_env(Env::default().filter_or("LOG_LEVEL", "info"));
```

I think if we just want to ignore the missing file this could be a better approach without the need of a new function